web/src/hooks/useColors.ts

```diff
@@ -1,10 +1,20 @@
 import { useColorModeValue } from "@chakra-ui/react";
+import { generateKeyPair } from "crypto";
```

Comment:
what is this? :D

Reply:
It is something that came with @magnusRBrok's changes in PR #22